### ISSUE TITLE
Call to a member function getUrl() on array

### ISSUE THREAD

**Issue by Afcyy** [marcreichel/igdb-laravel#74]
When I try to use function getUrl() I get the error message Call to a member function getUrl() on array;

### GamesController:
```
use MarcReichel\IGDBLaravel\Models\Game;

$games = Game::select(["*"])->with(['cover'])->get();
return GamesResource::collection($games);
```

### GamesResource:
```
use MarcReichel\IGDBLaravel\Enums\Image\Size;

public function toArray(Request $request): array
    {
        return [
            'cover' => $this->cover->getUrl(Size::COVER_BIG),
        ];
    }
```

**Comment by marcreichel** [comment]
I will look into this 👍🏼 Sorry for the inconveniences.

**Comment by marcreichel** [comment]
@Afcyy Fixed with version 4.0.0. It's a major release, because the minimum PHP version is bumped to PHP 8.1.

Please update to that release and check again.
